src/plugins/gui/fiatPlugin.tsx

```diff
@@ -193,7 +194,9 @@ export const executePlugin = async (params: {
     openExternalWebView: async (params): Promise<void> => {
       const { deeplinkHandler, providerId, redirectExternal, url } = params
       datelog(
-        `**** openExternalWebView ${url} deeplinkHandler:${deeplinkHandler}`
+        `**** openExternalWebView ${url} deeplinkHandler:${JSON.stringify(
+          deeplinkHandler
```

Comment:
Wtf... Why are we logging a function? The proper fix is to remove this deeplinkHandler from the log